CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.1.0]
+### Changed
+- Lint fix: fix for invalid syntax in index.js (#84) ([#84](https://github.com/MetaMask/eth-json-rpc-filters/pull/84))
```

Comment:
"lint fix" suggests that this was just a style issue, but this was a bug fix. This should the `eth_getFilterLogs` method, which was totally broken before

```suggestion
- Fix `eth_getFilterLogs` (#84) ([#84](https://github.com/MetaMask/eth-json-rpc-filters/pull/84))
```